websites/D/Discohook/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "150987570715688960",
+		"name": "Ninja"
+	},
+	"service": "Discohook",
+	"description": {
+		"en": "Discohook is a free tool that allows you to personalise your server to make your server stand out from the crowd."
+	},
+	"url": "discohook.org",
+	"version": "1.1.0",
```

New presences should be at `1.0.0`
```suggestion
	"version": "1.0.0",
```

---

This has been updated